### ISSUE TITLE
Convert samples to use shared flowgraph

### ISSUE THREAD

**Issue by philburk** [google/oboe#495]
Pull oscillators and other nodes from OboeTester.
Convert other nodes to flowgraph.

**Comment by atneya** [comment]
Note, this issue is waiting on #493

**Comment by philburk** [comment]
Nice to have a synth flowgraph library with oscillators and filters but it is a low priority.
